app/views/pages/home.html.haml

```diff
@@ -40,9 +40,8 @@
           %th= @weekly_bookings.count
           %td people booked within past week
         %tr.current-week-released
-          - total = @weekly_bookings.count
-          - inactive = @weekly_bookings.inactive.count
-          %th #{inactive} (#{present_percent(inactive, @weekly_bookings.count)})
+          - released = @weekly_releases.count
+          %th #{released} (#{present_percent(released, @active_bookings.count)})
```

Comment:
Should the percentage be calculated from people who are currently booked, as it is now?
Or should we calculate it from the people who were booked at the beginning of this week, as in:

```
bookings_one_week_ago = @active_bookings + released - booked_this_week
present_percent(released, bookings_one_week_ago)
```